tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -3242,18 +3242,20 @@ public function theUserShouldHaveSpaceWithRecipient(
 	}
 
 	/**
-	 * @When user :user downloads the space :spaceName using the WebDAV API
+	 * @When user :user tries to download the space :spaceName owned by user :owner using the WebDAV API
+	 * @When /^user "([^"]*)" (?:downloads|tries to download) the space "([^"]*)" using the WebDAV API$/
 	 *
 	 * @param string $user
 	 * @param string $spaceName
+	 * @param string $owner
 	 *
 	 * @return void
 	 *
 	 * @throws GuzzleException
 	 */
-	public function userDownloadsTheSpaceUsingTheWebdavApi(string $user, string $spaceName):void {
-		$space = $this->getSpaceByName($user, $spaceName);
-		$url = $this->featureContext->getBaseUrl() . '/archiver?id=' . $space["id"];
+	public function userDownloadsTheSpaceUsingTheWebdavApi(string $user, string $spaceName, string $owner = ''):void {
+		$space = $this->getSpaceByName($owner ?? $user, $spaceName);
```

Comment:
use ternary operator. you have empty string as default for `$owner` which will evaluate to true with `??` (nullish coalescing )
```suggestion
		$space = $this->getSpaceByName($owner ?: $user, $spaceName);
```